frontend/components/commons/forms/questions-form/Questions.component.vue

```diff
@@ -90,29 +90,21 @@ export default {
       const firstElement = focusable[0];
       const lastElement = focusable[focusable.length - 1];
 
-      const isShiftKeyPressed = e.shiftKey;
-
-      const isArrowDownPressed = e.key === "ArrowDown";
-      const isArrowUpPressed = e.key === "ArrowUp";
+      const isTabPressed = e.key == "Tab";
       const activeElementIsInForm = this.formWrapper.contains(
         document.activeElement
       );
       const isLastElementActive = document.activeElement === lastElement;
       const isFirstElementActive = document.activeElement === firstElement;
 
-      if (!activeElementIsInForm && isShiftKeyPressed && isArrowDownPressed) {
+      if (!activeElementIsInForm && isTabPressed) {
```

Comment:
Why don't you compare arrows anymore ?

Reply:
Here is only the to focus on form.
The code to go next/prev question with `shift+arrow` is in `Questions.component`